btree/btree.c

```diff
@@ -571,6 +573,38 @@ enum {
 	M0_NODE_COUNT = 100,
 };
 
+static struct m0_sm_state_descr btree_states[16] = {
```

Comment:
1. As, we will be having different state_descr for each operation, instead of btree_states, name  should be specific to operation
2. As, there are only 3 states, 16 should be replace by 3

Reply:
Reply for 1: Currently we are using only 3 states for create. As we go forward, this struct can be updated to include all possible transitions for the btree operations. Hence, there is no need to change the name. 
Refer to: be/tx_group_fom.c line 115 
Reply for 2: Yes, they can be reduced but we cant use 3. We have our "enum base_phase" which basically gives us 9 states to consider for btree though in create operation only 3 are used. So changing it to 9 will make more sense.